alignment/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'maven-publish' // only used for publishing to local maven
+    id 'java-gradle-plugin' // applies "java" plugin and adds gradleApi() to dependencies
+}
+
+group 'org.jboss.pme.alignment'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
+    compile 'org.apache.commons:commons-lang3:3.8.1'
+    compile 'commons-io:commons-io:2.6'
```

Comment:
Can we remove those dependencies until we actually need them?

Reply:
These dependencies are actually used.

Reply:
Ok cool, thanks!